pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/omstillingsstoenad/revurdering/RevurderingEndring.kt

```diff
@@ -0,0 +1,93 @@
+import no.nav.pensjon.etterlatte.maler.omstillingsstoenad.revurdering.OMSRevurderingEndringDTOSelectors.avkortingsinfo
+import no.nav.pensjon.etterlatte.maler.omstillingsstoenad.revurdering.OMSRevurderingEndringDTOSelectors.beregningsinfo
+import no.nav.pensjon.etterlatte.maler.omstillingsstoenad.revurdering.OMSRevurderingEndringDTOSelectors.erEndret
+import no.nav.pensjon.etterlatte.maler.omstillingsstoenad.revurdering.OMSRevurderingEndringDTOSelectors.etterbetalinginfo
+import no.nav.pensjon.etterlatte.maler.omstillingsstoenad.revurdering.OMSRevurderingEndringDTOSelectors.innhold
+
+data class OMSRevurderingEndringDTO(
+    override val innhold: List<Element>,
+    val avkortingsinfo: Avkortingsinfo,
+    val etterbetalinginfo: EtterbetalingDTO? = null,
+    val beregningsinfo: Beregningsinfo,
+    val erEndret: Boolean
+): BrevDTO
+
+@TemplateModelHelpers
+object RevurderingEndring : EtterlatteTemplate<OMSRevurderingEndringDTO>, Hovedmal {
+    override val kode: EtterlatteBrevKode = EtterlatteBrevKode.OMS_REVURDERING_ENDRING
+
+    override val template = createTemplate(
+        name = kode.name,
+        letterDataType = OMSRevurderingEndringDTO::class,
+        languages = languages(Bokmal, Nynorsk, English),
+        letterMetadata = LetterMetadata(
+            displayTitle = "Vedtak - Revurdering av omstillingsstønad",
+            isSensitiv = true,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VEDTAK,
+            brevtype = LetterMetadata.Brevtype.VEDTAKSBREV,
+        )
+    ) {
+        title {
+            text(
+                Bokmal to "Vi har ",
+                Nynorsk to "",
+                English to "",
+            )
+            showIf(erEndret) {
+                text(
+                    Bokmal to "endret",
+                    Nynorsk to "",
+                    English to "",
+                )
+            } orShow {
+                text(
+                    Bokmal to "vurdert",
+                    Nynorsk to "",
+                    English to "",
+                )
+            }
+            text(
+                Bokmal to " omstillingsstønaden din",
+                Nynorsk to "",
+                English to "",
+            )
+        }
+
+        outline {
+            includePhrase(Vedtak.BegrunnelseForVedtaket)
+
+            konverterElementerTilBrevbakerformat(innhold)
+
+            includePhrase(OMSInnvilgelse.Inntektsendring)
+            includePhrase(OMSInnvilgelse.Etteroppgjoer)
+            includePhrase(OMSFelles.MeldFraOmEndringer)
+            includePhrase(OMSFelles.DuHarRettTilAaKlage)
+            includePhrase(OMSFelles.HarDuSpoersmaal)
+        }
+
+        includeAttachment(beregningAvOmstillingsstoenad, beregningsinfo)
+        includeAttachment(informasjonOmOvergangsstoenad, avkortingsinfo)
+        includeAttachment(dineRettigheterOgPlikterOMS, avkortingsinfo)
+        includeAttachment(informasjonOmYrkesskade, avkortingsinfo)
```

Comment:
Skal alle disse ha `avkortingsinfo`?

Reply:
Bruker avkortingsinfo som attatchmentdata for å kunne kompilere. Bytter til innhold så gjør vi det likt som BP